docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -64,58 +37,87 @@ To apply the policies on a specific team use the `--policies-team` flag:
-- 3.7 Audit Software Inventory (Level 2): Scan systems on a monthly basis and determine the number of unauthorized pieces of software that are installed. Verify that if an unauthorized piece of software is found one month, it is removed from the system the next. Note: This can be accomplished via the Software Inventory feature provided by Fleet.
-- 5.2.3 Ensure Complex Password Must Contain Alphabetic Characters Is Configured (Level 2), 5.2.4 Ensure Complex Password Must Contain Numeric Character Is Configured (Level 2), 5.2.5 Ensure Complex Password Must Contain Special Character Is Configured (Level 2) and 5.2.6 Ensure Complex Password Must Contain Uppercase and Lowercase Characters Is Configured: The CIS macOS community has decided to not require the additional password complexity settings (Recommendations 5.3 - 5.6). Because of that, we have left the complexity recommendations as a manual assessment. Since there are a large amount of admins in the greater macOS world that do need these settings, we include both the guidance for the proper setting as well as probes for CIS-CAT to test.
-- 5.3.1 Ensure all user storage APFS volumes are encrypted (Level 1): In order to protect user data from loss or tampering volumes, carrying data should be encrypted.
-- 5.3.2 Ensure all user storage CoreStorage volumes are encrypted (Level 1): In order to protect user data from loss or tampering, volumes carrying data should be encrypted.
-- 6.2.1 Ensure Protect Mail Activity in Mail Is Enabled (Level 2): Email is routinely abused by attackers, spammers and marketers. The "Protect Mail Activity" control reduces risk by hiring the current IP address of your Mac and privately downloading remote content.
-- 6.3.2 Audit History and Remove History Items (Level 2): Old browser history becomes stale and the use or misuse of the data can lead to unwanted outcomes. Search engine results are maintained and often provide much more relevant current information than old website visit information.
-- 6.3.5 Audit Hide IP Address in Safari Setting (Level 2): Trackers can correlate your visits through various applications including websites and is a threat to your privacy. 
-
-Please refer to the "CIS Apple macOS 13.0 Ventura Benchmark v1.0.0 - 11-14-2022" PDF for descriptions and instructions on how to remediate.
+The following CIS benchmark checks cannot be automated and must be addressed manually:
+- 2.1.2 Audit App Store Password Settings
+- 2.3.3.12 Ensure Computer Name Does Not Contain PII or Protected Organizational Information
+- 2.6.6 Audit Lockdown Mode
+- 2.11.2 Audit Touch ID and Wallet & Apple Pay Settings
+- 2.13.1 Audit Passwords System Preference Setting
+- 2.14.1 Audit Notification & Focus Settings
+- 3.7 Audit Software Inventory
```

Comment:
Nit: We could mention that this check is solved by using Fleet. (We provide the software inventory.)
![Screenshot 2023-03-28 at 16 25 39](https://user-images.githubusercontent.com/2073526/228345941-3a761ed4-2806-4400-8206-b72425c4afda.png)

They can write policies to exclude specific software.

Reply:
E.g. a policy `Zoom app not installed` with the following query: `SELECT 1 WHERE NOT EXISTS(SELECT 1 FROM apps WHERE name like 'Zoom%');`.

But probably better to just use the Fleet API to read software and perform processing on the results.